src/attitude-estimators.hpp

```diff
@@ -42,6 +52,16 @@ class TriadAlgorithm : public AttitudeEstimationAlgorithm {
     Attitude Go(const Camera &, const Stars &, const Catalog &, const StarIdentifiers &);
 };
 
+/**
+ * A faster but not as accurate attitude estimator as the Davenport Q Algorithm.
```

Comment:
Equally as accurate as davenport q when using real-world floating point numbers, I recall that AHRS docs said that you usually can estimate the eigenvalue within the precision of a floating point number using newton raphson.